g2p/mappings/utils.py

```diff
@@ -86,7 +86,8 @@ def expand_abbreviations(data: str, abbs: Dict[str, List[str]], recursion_depth=
     """
     if recursion_depth > 10:
         raise exceptions.RecursionError(
-            "Too many levels of recursion in your abbreviation expansion. Check your abbreviations for circular references."
+            "Too many levels of recursion in your abbreviation expansion. "
```

Comment:
This and several other reformatting changes are just to appease flake8

Reply:
Note that although black will reformat things at "80-ish" columns we have lots and lots and lots of untouched legacy code with way more columns than that so I had to set the limit at 120.  Not sure if we want to go to the [recommended flake8 options for black](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8) or not.

Reply:
This was our old flake8 config in `setup.cfg` before your hatch PR, probably just want to go back to these settings?
```
[flake8]
ignore = E203, E302, E501, E999, W503
max-line-length = 88
max-complexity = 12
select = B,C,E,F,W,B9
```

Reply:
Notice E501 there, in particular. The fact we've been silencing it would explain why we have many screens of it that we've ignored so far.

Reply:
oh... yeah... well if you ignore E501 then max-line-length does nothing :)